src/components/schedule-page/validation-drawer/validation-drawer.component.tsx

```diff
@@ -67,3 +69,30 @@ export default function ValidationDrawerComponent(): JSX.Element {
     </div>
   );
 }
+
+export function SpanErrors(props: {
+  errors: ScheduleErrorMessageModel[] | undefined;
```

Comment:
Lepiej zdefiniować to jako interfejs, ponieważ może być tak, że potem ten interfejs stanie większy. 
I wtedy to będzie niewygodnie rozszerzać.